src/tests/ftest/rebuild/cascading_failures.yaml

```diff
@@ -12,7 +12,7 @@ timeout: 360
 server_config:
   name: daos_server
   servers:
-    targets: 8
+    targets: 1 #8, after fixed of DAOS-2799
```

Comment:
If we leave the comment here with no ticket to revisit this, I think chances are we are going to forget about this.
Can we 
1. create a test variant that will run with server target 8 and use either skipForTicket or cancelForTicket to skip the target=8 variant?
2. create a JIRA ticket to re-enable the target=8 test and use it to block DAOS-2799?

Reply:
Have created DAOS-6364 for target: 8 after DAOS-2799 resolved.

Reply:
Thanks. I think you need to modify yaml and python here fopr this test to be able to skip the target=8 variant using either cancelForTicket or skipForTicket, instead of using comments.

Reply:
Done.
Please see the new commit.